common/source/docs/common-advanced-configuration.rst

```diff
@@ -10,6 +10,7 @@ tuning options for the vehicle.
 .. toctree::
     :maxdepth: 1
 
+    Advanced Power Monitor Configuration<common-advanced-battery-setup>
```

Comment:
I'd probably name this one "Power Monitor Advanced Configuration" to keep the "Power " part at the front for alphabetical organisation.

By the way, we use "Module" and "Monitor" quite a bit so we might want to standardise on one or the other.